src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -315,6 +315,22 @@ static CaptureCallState captureCallState(String... capturedState) {
             return new LinkerOptions.CaptureCallStateImpl(set);
         }
 
+        /**
+         * {@return A linker option used to mark a function as trivial}
+         * <p>
+         * A 'trivial' function is a function that has an extremely short running time
+         * in all cases (similar to calling an empty function), and does not call back into Java.
```

Comment:
```suggestion
         * in all cases (similar to calling an empty function), and does not call back into Java (e.g. using an upcall stub).
```

Reply:
We should probably mention `getpid` somewhere as the poster child for trivial calls.